controls/package_spec.rb

```diff
@@ -74,13 +74,14 @@
   describe package(val_syslog_pkg) do
     it { should be_installed }
   end
+  only_if { os.name != 'fedora' }
```

Comment:
@shoekstra can we have here a short comment, explaining the reason for this `only_of` and fedora exclusion? (its not obvious for somebody, who does not know the context and problem here)